### PR TITLE
Update cross-references-divs.qmd

### PR DESCRIPTION
**Pull request by sgelzenleuchter**
Fix anchor link to section [Listings](https://quarto.org/docs/authoring/cross-references-divs.html#listings).

### CODE REVIEW

File: docs/authoring/cross-references-divs.qmd [hunk 1]
```diff
@@ -6,7 +6,7 @@ title: Cross-Reference Div Syntax
 
 ## Overview
 
-Cross-referenceable [figures](#figures), [tables](#tables) and [code listings](#code-listings) are known as *float* cross-references. Floats can appear in the rendered document at locations other than where they are defined, i.e. they float, and usually have captions.
```
Comment (cderv) [inline docs/authoring/cross-references-divs.qmd:9]: This anchor points to https://quarto.org/docs/authoring/cross-references.html#code-listings 

And so the id seems right `#code-listings`- the link at https://quarto.org/docs/authoring/cross-references.html#overview works ok for me.

**Review by cderv — COMMENTED** [review]
Thanks. 

Not sure about why this update is needed though... 🤔 

Can you give more context ?

File: docs/authoring/cross-references-divs.qmd (hunk 1 above)
Comment (cwickham) [reply docs/authoring/cross-references-divs.qmd:9]: The other links (`figures` and `tables`) point to sections on this page (`cross-reference-divs.qmd`) so `code listings` should too. But, the section is actually called `"Listings"` not `"Code Listings"` on this page. I think this change is required. I'll generate preview to check.

Comment (cderv) [reply docs/authoring/cross-references-divs.qmd:9]: Indeed the preview fixed the link. Thanks for the explanation ! 

And thanks @sgelzenleuchter !

**Pull request merged**